src/effects/VST/VSTControlMSW.h

```diff
@@ -11,7 +11,7 @@
 #ifndef AUDACITY_VSTCONTROLMSW_H
 #define AUDACITY_VSTCONTROLMSW_H
 
-#include <Windows.h>
+#include <windows.h>
```

Comment:
This is wrong.
Correct name is `Windows.h`, with the "W" letter capitalized.

Reply:
This is broken in MinGW for, like, always. However, this probably should be put under some ifdef, like `_MSC_VER`

Reply:
how is this wrong?
on Windows, the filesystems are case-insensitive, on Linux they generally are case-sensitive (building the linux kernel requires case-sensitive FS for example)

from my many years working with mingw, I have always seen `windows.h` with lowercase `w`.
case in point:
```
$ dpkg -L mingw-w64-common | grep indows.h
/usr/share/mingw-w64/include/windows.h
```

Reply:
That is not an issue for me, but this is a bug in MinGW. Because officially the header starts with capital W

Reply:
ah alright, apologies, I never used MSVC so I was not aware.

Reply:
@crsib how to proceed here, should this have a macro for different includes if MSVC?
it is not canonical, but lowercase windows.h works on MSVC because of case-insensitive file system.

Reply:
Maybe stick with `windows.h` and leave a comment about the MinGW bug?

Reply:
I'm fine with leaving it lowercase

@falkTX, I'm really sorry I haven't responded earlier

Reply:
> @falkTX, I'm really sorry I haven't responded earlier

no worries, I do not think there is a rush to merge this.

Reply:
If I can say my opinion, this is a false problem. As far as I know, the bug of names of the include files in W32API are known by long (long...) time. Usually, it is not a problem to use `window.h` instead of `Windows.h` or `wInDoWs.H`, but I think that if you want to do a safe cross compile for Windows on a linux machine, then you should put the include files elsewhere, into a partition with a case insensitive file system or into an image file and then mount it.

BTW, me too I'm building Audacity for Windows with MinGW: actually, there is an interesting advantage for doing this because, unlike official 32bit binaries, it can also run on Windows XP and NT (see issue #495).